### ISSUE TITLE
Raw expressions work with `insert` but not `insertWithRelated`

### ISSUE THREAD

**Issue by jbrantly** [Vincit/objection.js#103]
I'm trying to use a raw expression for an insert, ie:

```
Person
  .query()
  .insert({
    age: Person.query().avg('age'),
    firstName: Person.raw("'Jenni' || 'fer'")
  });
```

However, this does not work if using `insertWithRelated` instead of `insert`. Using `debug()` it seems like the property is simply not picked up (ignored).

**Comment by koskimas** [comment]
This seems like a clear bug. I'll fix this for the 0.5.0 release. I'll probably make an alpha/rc release soon that contains a fix for this. I'll keep you posted.

**Comment by koskimas** [comment]
Fixed in 0.5.0-rc.1. Run `npm install objection@next` to install.
